src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -415,6 +417,9 @@ export default class AlterationEnrichmentContainer extends React.Component<
             );
         }
 
+        const useInlineTypeSelectorMenu = !(window as any).frontendConfig
```

Comment:
we should always read the configuration off SiteConfig import, not from global

Reply:
I do not find the class SiteConfig anywhere.

Reply:
Sorry, AppConfig

Reply:
See for example around l386 of ResultsViewPage

Reply:
Ok, I have replaced all access to serverConfig via AppConfig.

Reply:
nudge